src/Voxel.MiddyNet/MiddyNetContext.cs

```diff
@@ -1,30 +1,36 @@
-﻿using System;
+﻿using Amazon.Lambda.Core;
+using System;
 using System.Collections.Generic;
-using Amazon.Lambda.Core;
 
 namespace Voxel.MiddyNet
 {
     public class MiddyNetContext
     {
         public ILambdaContext LambdaContext { get; private set; }
         public IDictionary<string, object> AdditionalContext { get; }
-        public List<Exception> MiddlewareExceptions { get; set; }
+        public List<Exception> MiddlewareBeforeExceptions { get; }
+        public List<Exception> MiddlewareAfterExceptions { get; }
+        public Exception HandlerException { get; set; }
```

Comment:
To be consistent with collections for exceptions in Before/After (`MiddlewareBeforeExceptions` & `MiddlewareAfterExceptions`), maybe we should use `MiddlewareHandlerException` instead of `HandlerException` as object name